var/spack/repos/builtin/packages/w3m/package.py

```diff
@@ -48,14 +48,17 @@ class W3m(AutotoolsPackage):
     patch('fix_redef.patch')
     patch('fix_gc.patch')
 
+    def patch(self):
+        filter_file('#define USE_EGD', '#undef USE_EGD', 'config.h.in')
```

Comment:
Is there any issue we can reference upstream for this patch?

Reply:
The same change has been applied by other package systems - the last release of w3m was in 2011.

I could add references to those, but it's hardly worth it :)

Reply:
In gentoo, EGD is disabled since 2016, most reports to disable it in w3m are from 2018 it seems.
https://gitweb.gentoo.org/repo/gentoo.git/commit/www-client/w3m?id=06852217367c5e858633f078aca05c6dc991688e

Reply:
Just add a comment with the Gentoo link on top of the filter file, then ping me for review. Thanks!

Reply:
A few comments about the current state of w3m in case someone looks at the package.py file and does not know the state of w3m by heart.